src/themes/brutalist.css

```
@@ -0,0 +1,832 @@
+  --wa-space-2xs: calc(var(--wa-space-base) * 0.25rem); /* 4px */
+  --wa-space-xs: calc(var(--wa-space-base) * 0.5rem); /* 8px */
+  --wa-space-s: calc(var(--wa-space-base) * 0.75rem); /* 12px */
+  --wa-space-m: calc(var(--wa-space-base) * 1rem); /* 16px */
+  --wa-space-l: calc(var(--wa-space-base) * 1.25rem); /* 20px */
+  --wa-space-xl: calc(var(--wa-space-base) * 1.5rem); /* 24px */
+  --wa-space-2xl: calc(var(--wa-space-base) * 2rem); /* 32px */
+  --wa-space-3xl: calc(var(--wa-space-base) * 3rem); /* 48px */
+
+  /**
+   * Borders
+   */
+  --wa-border-style: solid;
+
+  /* Border widths are designed to scale according to a single base value.
+     * The base value is intended for calculations and is not used by components directly. */
+  --wa-border-width-base: 0.0625;
+  --wa-border-width-s: calc(var(--wa-border-width-base) * 1rem);
+  --wa-border-width-m: calc(var(--wa-border-width-base) * 2rem);
+  --wa-border-width-l: calc(var(--wa-border-width-base) * 3rem);
+
+  --wa-form-controls-border-style: var(--wa-border-style);
+  --wa-form-controls-border-width: var(--wa-border-width-m);
+
+  --wa-panel-border-style: var(--wa-border-style);
+  --wa-panel-border-width: var(--wa-border-width-m);
+
+  /**
+   * Corners
+   */
+  /* Corners are designed to scale according to a single base value.
+   * The base value is intended for calculations and is not used by components directly. */
+  --wa-corners-base: 0.001;
+  --wa-corners-xs: calc(var(--wa-corners-base) * 0.75rem);
+  --wa-corners-s: calc(var(--wa-corners-base) * 1rem);
+  --wa-corners-m: calc(var(--wa-corners-base) * 2rem);
+  --wa-corners-l: calc(var(--wa-corners-base) * 3rem);
+
+  /* Semantic corner properties create specific shapes beyond the theme's preferred corner styles. */
+  --wa-corners-pill: 9999px;
+  --wa-corners-circle: 50%;
+  --wa-corners-sharp: 0;
+
+  /**
+   * Focus
+   */
+  --wa-focus-ring-style: solid;
+  --wa-focus-ring-width: 0.1875rem; /* 3px */
+  --wa-focus-ring: var(--wa-focus-ring-style) var(--wa-focus-ring-width)
+    color-mix(in oklab, var(--wa-color-focus) 98%, transparent);
+  --wa-focus-ring-offset: 0.0625rem; /* 1px */
+
+  /**
+   * Links
+   */
+  --wa-link-decoration-default: underline var(--wa-color-neutral-border-highlight) dotted;
+  --wa-link-decoration-hover: underline;
+
+  /**
+   * Z-index
+   */
+  --wa-z-index-drawer: 700;
+  --wa-z-index-dialog: 800;
+  --wa-z-index-dropdown: 900;
+  --wa-z-index-alert-group: 950;
+  --wa-z-index-tooltip: 1000;
+
+  /**
+   * Transitions
+   */
+  --wa-transition-normal: 250ms;
+  --wa-transition-fast: 150ms;
+  --wa-transition-faster: 50ms;
+
+  /**
+   * Component groups
+   */
+  --wa-form-controls-background: var(--wa-color-surface-default);
+
+  --wa-form-controls-corners: var(--wa-corners-s);
+
+  --wa-form-controls-activated-color: var(--wa-color-neutral-spot);
+  --wa-form-controls-resting-color: var(--wa-color-neutral-border-subtle);
```

```suggestion
  --wa-form-controls-resting-color: var(--wa-color-neutral-border-highlight);
```

Rather than overriding the border colors for `wa-input`, `wa-select`, and `wa-checkbox`, we could use this property, which covers our inactive/deselected/unfocused color for inputs.